.circleci/config.yml

```diff
@@ -205,9 +205,20 @@ workflows:
           docker_tag: dev-snapshot
           requires:
             - test_cli_staging
+      - codacy/sbt:
+          name: publish_cli_jar
```

Comment:
publish where? I was kinda expecting some sort of "publish" command, how will this become available?

Reply:
You're right, the naming is not the best. It is building the CLI runnable jar.
What do you think about renaming it to `build_cli_jar`?

Reply:
build_cli_jar or even build_jar seem good to me 👍